.github/workflows/build-nightly-images.yaml

```
@@ -0,0 +1,141 @@
+      matrix-config:
+        description: 'JSON string containing the matrix configuration'
+        required: true
+        type: string
+      branch:
+        description: 'Branch to build from'
+        required: true
+        type: string
+    outputs:
+      dependent-jobs:
+        description: 'Dependent jobs to build next'
+        value: ${{ jobs.build-images.outputs.dependent-jobs }}
```

actionlint (1.7.10)

[error] property "dependent-jobs" is not defined in object type {}

(expression)